src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -174,6 +174,10 @@ Plugin::Plugin() {
     for (const auto& device : m_device_map) {
         m_configs_map.insert({device.first, ExecutionConfig(ov::device::id(device.first))});
     }
+
+    // Set common info for compiled_model_runtime_properties
+    auto& ov_version = ov::get_openvino_version();
+    m_compiled_model_runtime_properties["OV_VERSION"] = ov_version.buildNumber;
```

Comment:
It looks like `m_compiled_model_runtime_properties` is common in cpu-plugin and gpu-plugin.
How about having this only one in any higher level?
Other than this, it looks good to me in the gpu-plugin side.

Reply:
Good point! But this data only is available for hardware plugin, and not be used for meta plugin. Higher level will be for all plugins.

Reply:
Ah! understood. Thank you!